code/obj/artifacts/artifactprocs.dm

```diff
@@ -166,6 +166,64 @@
 		src.UpdateOverlays(null, "activated")
 	A.effect_deactivate(src)
 
+/obj/proc/Artifact_emp_act()
+	if (!src.ArtifactSanityCheck())
+		return
+	src.ArtifactStimulus("elec", 800)
+	src.ArtifactStimulus("radiate", 3)
+
+/obj/proc/Artifact_blob_act(var/power)
+	if (!src.ArtifactSanityCheck())
+		return
+	src.ArtifactStimulus("force", power)
+	src.ArtifactStimulus("carbtouch", 1)
+
+/obj/proc/Artifact_reagent_act(var/reagent_id, var/volume)
+	if (!src.ArtifactSanityCheck())
+		return
+	var/datum/artifact/A = src.artifact
+	src.ArtifactStimulus(reagent_id, volume)
```

Comment:
It was in the original code for handling reagent interactions with artifacts.

I assumed it handled some kind of weird and specific edge case, so I left it in.

Otherwise, yeah, I'll absolutely remove it.